specutils/tests/test_loaders.py

```diff
@@ -599,27 +599,47 @@ def test_tabular_fits_header(tmp_path):
     disp = np.linspace(1, 1.2, 21) * u.AA
     flux = np.random.normal(0., 1.0e-14, disp.shape[0]) * u.Jy
     hdr = fits.header.Header({'TELESCOP': 'Leviathan', 'APERTURE': 1.8,
-                              'OBSERVER': 'Parsons', 'NAXIS': 1, 'NAXIS1': 8})
+                              'OBSERVER': 'Parsons'})
 
     spectrum = Spectrum1D(flux=flux, spectral_axis=disp, meta={'header': hdr})
     tmpfile = str(tmp_path / '_tst.fits')
     spectrum.write(tmpfile, format='tabular-fits')
 
+    print(f"spetrum meta: \n {repr(spectrum.meta['header'])}")
     # Read it in and check against the original
     with fits.open(tmpfile) as hdulist:
-        assert hdulist[0].header['NAXIS'] == 0
```

Comment:
Is this no longer working? Would be worrying, because an "empty" PrimaryHDU should always have `NAXIS = 0`!

Reply:
I'll put it back in!